doc/users/whats_new.rst

```diff
@@ -19,6 +19,15 @@ New in matplotlib 2.0
 
    matplotlib 2.0 supports Python 2.7, and 3.4+
 
+The behavior of AxesStack re-add key changes
```

Comment:
Agreed that this should be moved to the api_changes and cleaned up (see @QuLogic's comment below).